distributed/diagnostics/progress.py

```diff
@@ -291,3 +294,83 @@ def transition(self, key, start, finish, *args, **kwargs):
     def restart(self, scheduler):
         self.all.clear()
         self.state.clear()
+
+
+class GroupTiming(SchedulerPlugin):
+    """Keep track of high-level timing information for task group progress"""
+
+    name = "group-timing"
+
+    def __init__(self, scheduler):
+        scheduler.add_plugin(self)
```

Comment:
This is the line that is causing the double registering of the plugin hence the warning. Notice we are registering the plugin on https://github.com/ian-r-rose/distributed/blob/8b53b447330704b168c85de6b967868c605f1789/distributed/dashboard/components/scheduler.py#L2571-L2572
```suggestion

```

Reply:
Good catch @ncclementi , thanks!